tests/test_released_data.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+from sqlalchemy.sql.expression import func
+
+from gdc_ng_models.models import released_data
+
+
+@pytest.fixture
+def fake_released_data(create_released_data_db, db_session):
+    def helper(
+        name="name", code="code", is_controlled=False, data_type="cnv", is_open=False,
+    ):
+        node = released_data.ReleasedData(
+            program_name=name,
+            project_code=code,
+            is_controlled=is_controlled,
+            data_type=data_type,
+            is_open=is_open,
+        )
+
+        db_session.merge(node)
+        return node
+
+    return helper
+
+
+@pytest.fixture
+def fake_released_log(create_released_data_db, db_session, request):
+    def helper(
+        name="name", code="code", release_number="1", data_type="cnv", is_open=False,
+    ):
+        node = released_data.ReleasedDataLog(
+            program_name=name,
+            project_code=code,
+            release_number=release_number,
+            data_type=data_type,
+            is_open=is_open,
+        )
+        db_session.merge(node)
+        db_session.commit()
+        return node
+
+    def tearDown():
```

Comment:
There's another `tearDown` here that I don't think is needed... doesn't really hurt, I guess.